lib/algos/list/tools.js

```diff
@@ -31,9 +34,29 @@ function inc(str) {
             String.fromCharCode(str.charCodeAt(str.length - 1) + 1)) : str;
 }
 
+function listingParamsMasterKeysV0ToV1(v0params) {
+    const v1params = Object.assign({}, v0params);
```

Comment:
Just a preference thing but the spread operator should work here.
`const v1Params = {...v0params};` less typing but same result.

Reply:
Our linter config does not like this construct, so will leave it as is.

Reply:
Oh, might have an older version of eslint. Oh well.